### PR TITLE
[docs/enhancement] - supercollider/HelpSource/Referece/emacseditor.schelp - improving grammar, et. al

### PR DESCRIPTION
**Pull request by tem44**
[docs/enhancement] - supercollider/HelpSource/Referece/emacseditor.schelp - improving grammar, et. al

### CODE REVIEW

File: HelpSource/Reference/EmacsEditor.schelp [hunk 1]
```diff
@@ -131,7 +131,7 @@ Setting the mark always makes an entry on the current buffer’s mark ring. If y
 
 ‘C-x C-SPC’ (‘pop-global-mark’) jumps to the buffer and position of the latest entry in the global mark ring. It also rotates the ring, so that successive uses of ‘C-x C-SPC’ take you to earlier and earlier buffers.
 
-section:: Customsing Emacs' behaviour
+section:: Customizing Emacs' behaviour
```
Comment (alln4tural) [inline HelpSource/Reference/EmacsEditor.schelp]: This mixes US and UK orthography now, should probably be behavior

File: HelpSource/Reference/EmacsEditor.schelp [hunk 2]
```diff
@@ -193,7 +193,7 @@ code::
 
 subsection:: Recent files menu
 
-Recentf is a minor mode that builds a list of recently opened files. This list is is automatically saved across Emacs sessions. You can then access this list through a menu. Put this in your `~/.emacs’:
```
Comment (alln4tural) [inline HelpSource/Reference/EmacsEditor.schelp]: That was not a typo, the mode is called recentf

**Comment by tem44** [comment]
all fixed. please proceed with double checking. thanks

**Comment by brianlheim** [comment]
@snappizz For minor things like this let's just squash and merge, OK? We've done that in the past and AFAICT you were asking for this branch to be rebased into a single commit anyway.

**Pull request merged**